### PR TITLE
Implement ticket refresh management for AB#14507.

### PR DESCRIPTION
**Pull request by BrianMaki**
# Implements [AB#14507](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/14507) and implements [AB#14511](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/14511)

## Description

Implements ticket refresh management:

- Updated service by adding log statements and replacing Ticket with Void in remove ticket
- Added handle ticket action to waitlist store to process ticket returned from create and check in
- Add functional tests
- Updated appsettings.Development.json by pointing waiting queue endpoint (Ticket service) to dev instead of localhost
- Updated login in command.js in functional tests - increased timeout time to get cookies and removed check header dropdown button as that will not be shown if queued or busy pages are displayed.

**Login and ticket is created with Processed status and then checks in successfully:**

**Unable to access login because there are no tickets available and queue is not accessible:**

Screenshot 2022-12-16 at 11 17 31 PM

**Login with Queue but after waiting is eventually logged in:**

Screenshot 2022-12-16 at 11 18 16 PM

Screenshot 2022-12-16 at 11 20 10 PM

**Functional test results:**

Screenshot 2022-12-17 at 10 12 22 PM

## Testing

- [ ] Unit Tests Updated
- [x] Functional Tests Updated
- [x] Not Required

## UI Changes

None

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

### CODE REVIEW

**Review by rayking29 — APPROVED** [review]

File: Apps/WebClient/src/appsettings.Development.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
         "Laboratory": "http://localhost:3004/",
         "Medication": "http://localhost:3003/",
         "Patient": "http://localhost:3002/",
-        "Ticket": "http://localhost:8000/"
+        "Ticket": "https://tickets.healthgateway.gov.bc.ca/"
```
Comment (ytqsl) [inline Apps/WebClient/src/appsettings.Development.json:52]: Is this change intentional or should be reverted?

**Review by ytqsl — APPROVED** [review]

File: Apps/WebClient/src/appsettings.Development.json (hunk 1 above)
Comment (BrianMaki) [reply Apps/WebClient/src/appsettings.Development.json:52]: I spoke with @sslaws.  This was intentional.  WaitingQueueWeb is a project outside of HealthGateway. WaitingQueueWeb will need to be brought in locally in order for local dev to work. This will be sorted out later.
